src/Admin/AbstractAdmin.php

```diff
@@ -566,12 +573,7 @@ abstract class AbstractAdmin implements AdminInterface, DomainObjectInterface, A
      */
     private $filterPersister;
 
-    /**
-     * @param string $code
-     * @param string $class
-     * @param string $baseControllerName
-     */
-    public function __construct($code, $class, $baseControllerName)
+    public function __construct(string $code, string $class, string $baseControllerName)
```

Comment:
It's a BC-break

Reply:
I guess no. At least, there are no test cases covering this method with other types.

Reply:
> It's a BC-break

👍 This is a break, some one could use null or other types

Reply:
Ok. I'll be adding deprecation notices if other types are used instead.
IMO. the only argument that should accept `string|null` and could be optional is `$baseControllerName`, which is the only one associated to a property with a setter. Do you agree?

Reply:
Deprecations were added. Please let me know if they seem fine for you.